spec/03-plugins/13-cors/02-schema_spec.lua

```diff
@@ -30,7 +30,27 @@ describe("cors schema", function()
 
       assert.falsy(ok)
       assert.equals("'invalid_**regex' is not a valid regex",
-                    err.config.origins[2])
+        err.config.origins[2])
     end)
   end)
 end)
+
+describe("methods in cors schema", function()
+  for _,method in ipairs({"HEAD", "GET", "POST", "PUT", "PATCH", "DELETE", "OPTIONS", "TRACE", "CONNECT"}) do
```

Comment:
style: missing a space after `,` and in the table definition (`{ "HEAD", ... }`).